packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,56 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// Here, we will use a [Shortcuts] and [Actions] widget to add and remove from a counter.
```

Comment:
Sorry, I should have been a little more specific. It should be in the dartpad :) Like this:
```dart
/// {@tool dartpad --template=stateful_widget_scaffold_center}
///
/// Here, we will use a [Shortcuts] and [Actions] widget to add and remove from a counter. 
/// This can be done by creating a child widget that is focused and pressing the logical key 
/// sets that have been defined in [Shortcuts] and defining the actions that each key set 
/// performs.
/// 
/// ```dart imports
/// ....
```

Reply:
Sure, no issues! 😄